index.js

```diff
@@ -1 +1,224 @@
-console.log('Live reloading')
\ No newline at end of file
+const startButton = document.querySelector('.startButton');
+
+const restart = document.querySelector(".restartButton");    // US7 Button.
+
+const redButton = document.querySelector('.redB');
+const greenButton = document.querySelector('.greenB');
+const blueButton = document.querySelector('.blueB');
+const yellowButton = document.querySelector('.yellowB');
+
+const textLevel = document.querySelector('.levelLabel');
+
+let gameSeqnc = [];
+let playerSeqnc = [];
+let round = 1;
+const MAX_ROUNDS = 20;
+let hardmode = false;
+
+let text = '';                                                  // US8 text.
+
+/* Event listener waiting for User to click START button. */
+startButton.addEventListener('click', () => {
+  removeAllActiveListeners();
+
+  if (hardmode) {
+    hardRun();
+  } else {
+    normalRun();
+  }
+});
+
+/* Function that runs a normal mode run of the game. */
+const normalRun = () => {
+  /* Function that gets the sequence that User will try to copy. */
+  gameSeqnc = fillGameSeqnc();
+
+  playerSeqnc = [];
+  round = 1;
+  updateLevel(round);
+
+  /* Function that shows the current round game-sequence. */
+  showCurrentGameSeqnc();
+
+  /* Event listeners waiting for User to click any of the four buttons. */
+  redButton.addEventListener('click', handleRedButton);
+  greenButton.addEventListener('click', handleGreenButton);
+  blueButton.addEventListener('click', handleBlueButton);
+  yellowButton.addEventListener('click', handleYellowButton);
+  restart.addEventListener('click', restarting);            // US7 restart event listener.
+};
+
+const hardRun = () => {};
+
+/* Function that returns the sequence that User will try to copy. */
+const fillGameSeqnc = () => {
+  const gameSeqnc = [];
+
+  while (gameSeqnc.length < 20) {
+    const newStep = rng(4);
+
+    switch (newStep) {
```

Comment:
So Gabriel, I have read about the enum datatype in other languages, we don't have one in JavaScript but we could use comments to document it and equal a number to a color in a comment instead of doing this switch.